Osiris/GUI.cpp

```diff
@@ -411,11 +411,15 @@ void GUI::renderBacktrackWindow(bool contentOnly) noexcept
         ImGui::Begin("Backtrack", &window.backtrack, windowFlags);
     }
     ImGui::Checkbox("Enabled", &config->backtrack.enabled);
+    ImGui::SameLine();
+    ImGui::Checkbox("Extend with fake ping", &config->backtrack.fakeLatency);
     ImGui::Checkbox("Ignore smoke", &config->backtrack.ignoreSmoke);
+    ImGui::SameLine();
     ImGui::Checkbox("Recoil based fov", &config->backtrack.recoilBasedFov);
-    ImGui::PushItemWidth(220.0f);
-    ImGui::SliderInt("Time limit", &config->backtrack.timeLimit, 1, 200, "%d ms");
```

Comment:
make better like this
```cpp
ImGui::SliderInt("Time limit", &config->backtrack.timeLimit, 1, config->backtrack.fakeLatency ? 400 : 200, "%d ms");
```

Reply:
Would this fix also change the config value like my code does when snapping from 400 to 200, 
or only change the visual GUI output onscreen? 
like =
`if (config->backtrack.timeLimit >= 201) { config->backtrack.timeLimit = 200; }`

ill test myself anyway now